### PR TITLE
Update information on API keys

### PR DESCRIPTION
**Pull request by zwalchuk**
Include a section on access tokens in the account management page, with both session tokens and named API keys.

Change-type: patch

### CODE REVIEW

File: pages/learn/manage/account.md [hunk 1]
```diff
@@ -8,6 +8,44 @@ title: Account management
...
+
+Session tokens are retrieved from the *Preferences* page, and they can be refreshed with the [API][api-refresh]. These tokens expire after seven days, and they cannot be revoked.
+
+### API keys
+
+API keys are named tokens that do not expire and can be revoked as needed. To create a new API key, make sure you are in the *Access tokens* tab of the *Preferences* page, then select *Create API key*:
+
+<img src="/img/common/preferences/create_api_key.png" width="80%">
+
+You'll see a required field for *Token name*, as well as an optional field for *Token description*:
```
Comment (alisondavis17) [inline pages/learn/manage/account.md]: Do we have any requirements around Token name that are worth documenting (i.e. length, whitespace, special chars)?

File: pages/learn/manage/account.md [hunk 2]
```diff
@@ -8,6 +8,44 @@ title: Account management
...
+
+After you close the dialog, you'll see your API key in the list, complete with name, date of creation, and description:
+
+<img src="/img/common/preferences/api_key_list.png" width="80%">
+
+To revoke one or more API keys, select the boxes to the left of the tokens you wish to remove, then click *Delete selected*:
+
+<img src="/img/common/preferences/api_key_delete.png" width="80%">
+
+API keys can also be generated using the API, [CLI][cli-keys], and [Node.js][node-sdk-keys] and [Python][python-sdk-keys] SDKs.
```
Comment (alisondavis17) [inline pages/learn/manage/account.md]: can they also be revoked using the API/CLI/SDK?

File: pages/learn/manage/account.md (hunk 1 above)
Comment (zwalchuk) [reply pages/learn/manage/account.md]: Good question for @thgreasi

Comment (thgreasi) [reply pages/learn/manage/account.md]: I'm naming them based on the scope that they will be used. eg for the CLI, for devices of application X.
Other than these, I can't think of any other suggestion. Maybe we could check what GH docs mention for their API keys.

File: pages/learn/manage/account.md (hunk 2 above)
Comment (alisondavis17) [reply pages/learn/manage/account.md]: @thgreasi @pimterry ?

**Pull request merged**